quantum/process_keycode/process_clicky.c

```diff
@@ -34,8 +39,8 @@ void clicky_play(void) {
 #ifndef NO_MUSIC_MODE
   if (music_activated || midi_activated) return;
 #endif // !NO_MUSIC_MODE
-  clicky_song[0][0] = 2.0f * clicky_freq * (1.0f + clicky_rand * ( ((float)rand()) / ((float)(RAND_MAX)) ) );
-  clicky_song[1][0] = clicky_freq * (1.0f + clicky_rand * ( ((float)rand()) / ((float)(RAND_MAX)) ) );
+  clicky_song[1][0] = 2.0f * clicky_freq * (1.0f + clicky_rand * ( ((float)rand()) / ((float)(RAND_MAX)) ) );
+  clicky_song[2][0] = clicky_freq * (1.0f + clicky_rand * ( ((float)rand()) / ((float)(RAND_MAX)) ) );
```

Comment:
The avalanche of type casts and parentheses isn't needed here, just `(rand() / (float)RAND_MAX)` will do.

Other than that, LGTM. 👍

Reply:
@vomindoraan why the change from 0 -> 1 and 1 -> 2

Reply:
Presumably he did that because he inserted the delay as note `[0]`, so what used to be the first note of the melody is now `[1]` and the second is `[2]`.

Reply:
yeah. 

Now if silence notes worked properly ....